lib/openlogi/api/items.rb

```diff
@@ -0,0 +1,33 @@
+require "openlogi/api/endpoint"
+
+module Openlogi
+  module Api
+    class Items < Endpoint
+      def resource_class
+        Openlogi::Item
+      end
+
+      def find(id, stock: false)
+        params = stock ? { stock: stock } : {}
+        perform_request_with_object(:get, "items/#{id}", params)
+      end
+
+      def all(stock: false)
+        params = stock ? { stock: stock } : {}
+        perform_request_with_objects(:get, "items", params)
+      end
+
+      def update(id, params)
+        perform_request_with_object(:put, "items/#{id}", params)
+      end
+
+      def create(params)
+        perform_request_with_object(:post, "items", params)
```

Comment:
I'm hesitating a bit about this, but if we wrapped `params` here with `resource_class.new(params)` we could validate parameters sent to openlogi with the same (hashie) model we use to wrap responses, so we could e.g. add required parameters, stuff like that.